client/command/extensions/load.go

```diff
@@ -525,6 +555,20 @@ func runExtensionCmd(cmd *cobra.Command, con *console.SliverClient, args []strin
 	}
 	bd := sha256.Sum256(extdata)
 	name := hex.EncodeToString(bd[:])
+	if isBOF {
+		//if we are using a bof, we are actually calling the coffloader extension - so get the file from dep ref and use that shasum
+		dep, err := loadedExtensions[ext.DependsOn].getFileForTarget(goos, goarch)
```

Comment:
Same as above, it's worth checking that `loadedExtensions[ext.DependsOn]` exists before calling `getFileForTarget()` on the result.